test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
```

Comment:
I'd suggest renaming this test to `parse.js` or maybe `css-tree.js` since it's not doing validation akin to `WebIDL2.validate`.